.github/workflows/docs.yaml

```diff
@@ -0,0 +1,46 @@
+name: Deploy Docs
+
+on:
+  push:
```

Comment:
TODO. Need to limit this to main before merging

Reply:
Does this still need to be done?

Reply:
This was added temporarily in order to have the served Github page be available during the review process. I've now limited the branch to `main`